File: Jaewon/Level.1/23) 이상한 문자 만들기/23.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
    string answer = "";
    int idx = 1;
    for (int i = 0; i < s.size(); i++, idx++)
    {
        if (s[i] == ' ')
        {
            idx = 0;
            answer += " ";
        }
        else
            idx % 2 != 0 ? answer += toupper(s[i]) : answer += tolower(s[i]);
    }

    return answer;
}

File: Jaewon/Level.1/23) 이상한 문자 만들기/23.js
function solution(s) {
    var answer = "";
    let whiteSpace = 0;
    for(let i = 0; i < s.length; i++){
        if(s.charAt(i) === ' '){
            answer += ' ';
            whiteSpace = 0;
        } else {
            answer += whiteSpace%2 != 0 ? s.charAt(i).toLowerCase() : s.charAt(i).toUpperCase();
            whiteSpace ++;
        }     
    }
    return answer;
}